tsconfig.json

```diff
@@ -1,37 +1,11 @@
 {
     "$schema": "https://json.schemastore.org/tsconfig",
+    "extends": "expo/tsconfig.base",
     "compilerOptions": {
-        "target": "esnext",
-        "module": "commonjs",
         "types": ["react-native", "jest"],
-        "lib": [
-            "DOM",
-            "es2019",
-            "es2020.bigint",
-            "es2020.date",
-            "es2020.number",
-            "es2020.promise",
-            "es2020.string",
-            "es2020.symbol.wellknown",
-            "es2021.promise",
-            "es2021.string",
-            "es2021.weakref",
-            "es2022.array",
-            "es2022.object",
-            "es2022.string",
-            "ES2021.Intl",
-            "ES2023.Array"
-        ],
-        "allowJs": true,
-        "checkJs": false,
-        "jsx": "react-native",
-        "noEmit": true,
         "isolatedModules": true,
         "strict": true,
-        "moduleResolution": "node",
-        "resolveJsonModule": true,
         "allowSyntheticDefaultImports": true,
-        "skipLibCheck": true,
```

Comment:
Are you sure this is something we want to remove? I know that we will be more typesafe without it, but I'm not sure if it's a good call in project with so many dependencies

Reply:
I removed this one as we extend expo config which already has this option, so no real changes here